File: backend/quick_set.py
import sys
import os
from mongoengine import connect
from dotenv import load_dotenv

sys.path.append(os.path.dirname(os.path.abspath(__file__)))
from models import User

load_dotenv()
mongo_uri = os.getenv("MONGO_URI", "mongodb://localhost:27017/unirank")
connect(host=mongo_uri)

user = User.objects(email="pes1ug25cs393@stu.pes.edu").first()
if user:
    user.role = "student"
    user.save()
    print("Successfully set role to student.")
else:
    print("User not found.")

File: backend/src/scripts/quickSet.js
/**
 * scripts/quickSet.js
 * Quick script to update a user's role.
 */

import dotenv from 'dotenv';
import connectDB from '../config/db.js';
import { User } from '../models/index.js';
import mongoose from 'mongoose';

dotenv.config();

async function main() {
  await connectDB();
  const user = await User.findOne({ email: 'pes1ug25cs393@stu.pes.edu' });
  if (user) {
    user.role = 'student';
    await user.save();
    console.log('Successfully set role to student.');
  } else {
    console.log('User not found.');
  }
  await mongoose.disconnect();
}

main().catch((err) => {
  console.error(err);
  process.exit(1);
});
